packages/client/src/app/components/comments/comments.component.ts

```diff
@@ -17,7 +17,6 @@ import { CommentFormComponent } from './comment-form/comment-form.component';
 export class CommentsComponent implements OnInit {
   @Input() itemId: string; // The ID of the blog/work/newspost
   @Input() itemKind: string; // The kind of comments thread it is, between blogs/works/newsposts
```

Comment:
`itemKind` can (and probably should) be an enum

Reply:
solved with the latest commit